sycl/test-e2e/format.py

```diff
@@ -201,6 +201,9 @@ def get_extra_env(sycl_devices):
             if "level_zero:gpu" in sycl_devices and litConfig.params.get("ur_l0_debug"):
                 extra_env.append("UR_L0_DEBUG={}".format(test.config.ur_l0_debug))
 
+            if "level_zero:gpu" in sycl_devices:
+                extra_env.append("UR_ENABLE_LAYERS=UR_LAYER_PARAMETER_VALIDATION")
```

Comment:
I don't think this should be limited to only being enabled on `level_zero:gpu`.

Reply:
Also, I'm unsure if this the correct place to set environment variables for an e2e run.